src/AppBundle/Repository/ProcurationProxyRepository.php

```diff
@@ -49,7 +49,7 @@ public function findManagedBy(Adherent $procurationManager, int $page, int $perP
         $this->addAndWhereManagedBy($qb, $procurationManager);
 
         if ($filters) {
```

Comment:
I would use a `$filter->hasQuery()` method so the filter can say if it has values to give to the builder.